typescript/sdk/src/consts/chainConnectionConfigs.ts

```diff
@@ -151,7 +151,7 @@ export const moonbeam: IChainConnection = {
     1284,
```

Comment:
just noticed the url above this has trailing spaces, might as well rm while we're here